content/sensu-go/5.15/getting-started/enterprise.md

```diff
@@ -27,25 +28,25 @@ See the [products page][3] for a complete feature comparison.
 
 ### Contact us for a free trial
 
-For a personalized demo and free trial of license-activated features in Sensu Go, [contact the Sensu sales team][1].
+For a personalized demo and free trial of commercial features in Sensu Go, [contact the Sensu sales team][1].
```

Comment:
Hm, the trial doesn't really apply to the commercial features anymore, rather the entity limit. Maybe something more about the scalability a higher entity limit in your trial can offer:
```suggestion
For a personalized demo and free trial of commercial features at scale in Sensu Go, [contact the Sensu sales team][1].
```